README.md

```diff
@@ -1,25 +1,48 @@
 # noir_base64
 
-A library to encode ASCII into Base64 and decode Base64 into ASCII
+A library to encode ASCII into Base64 and decode Base64 into ASCII.
 
-# Usage
+## Dependencies
 
-### `fn base64_encode`
-Takees an input byte array of ASCII characters and produces an output byte array of base64-encoded characters. The 6-bit base64 characters are packed into a concatenated byte array (e.g. 4 bytes of ASCII produce 3 bytes of encoded Base64)
+- Noir ≥v0.34.0
+- Compatible version of a proving backend, eg Barretenberg
```

Comment:
Let's remove references to the backend here. If we push people towards the Nargo docs then that can teach them about the need for a proving backend.

We're going to get a lot of duplicated documentation which will fall out of sync otherwise.